### DISCUSSION TITLE
Progress images with API generation gone

### DISCUSSION THREAD

**Discussion by ohnosharks** [vladmandic/sdnext#2691] [Q&A] [answered]
I used to be able to get progress images via the `/sdapi/v1/progress` endpoint, but the `current_image` value is now always null when generating with the txt2img endpoint. Progress images are still showing in the web UI and generations from the UI also show a current_image image in progress endpoint.

I've been just omitting the skip_current_image parameter in my requests, but setting it to false doesn't change anything. 

Not sure if bug or I missed something somewhere. (Windows 10, GTX1080Ti, master branch, SD 1.5)

**Comment by vladmandic** [comment]
i just tested, i'm getting image via progress api.

**Reply by ohnosharks** [reply]
Oh, weird. Is there a setting for the `/sdapi/v1/txt2img` endpoint that can disable them somehow? It's only when the progress api job value is `api-txt2img` that I'm not getting the image, from UI is fine. I'm testing with just the Docs' "Try it out" function.

**Reply by vladmandic** [reply, marked as answer]
ah, yes, progress image updates are currently skipped if generate is triggered via api.
i'll change that, it makes no sense - if user wants progress, he should get it.

**Reply by ohnosharks** [reply]
Ah I see – yes, thank you, that would be great!

**Reply by vladmandic** [reply]
added in dev, will be in the next release.
